PyRDF/backend/Dist.py

```diff
@@ -389,7 +389,10 @@ def reducer(values_list1, values_list2):
         if not self.nentries:
             # Fall back to local execution
             # if 'nentries' is '0'
-            return Local.execute(generator)
+            print("No entries in the Tree, falling back to local execution!")
```

Comment:
This could be a `UserWarning` instead of a plain `print`, similar to what we do when the number of partitions is greater than the number of clusters in a dataset.